services/src/pro/datasets/postgres.rs

```diff
@@ -504,6 +504,7 @@ where
 }
 
 #[async_trait]
+#[allow(dead_code)]
```

Comment:
maybe delete this trait together with DatasetMetaData struct

Reply:
[09346fc](https://github.com/geo-engine/geoengine/pull/973/commits/09346fcdb7960b7a52a8ecfce637354cb00950f8)